src/BizHawk.BizInvoke/BizExvoker.cs

```diff
@@ -67,8 +67,7 @@ public DelegateStorage(Type type)
 
 				foreach (var a in methods)
 				{
-					MethodBuilder unused;
-					var delegateType = BizInvokeUtilities.CreateDelegateType(a.Info, a.Attr!.CallingConvention, typeBuilder, out unused).CreateType()!;
+					var delegateType = BizInvokeUtilities.CreateDelegateType(a.Info, a.Attr!.CallingConvention, typeBuilder, out var unused).CreateType()!;

```

Comment:
```csharp
var delegateType = BizInvokeUtilities.CreateDelegateType(a.Info, a.Attr!.CallingConvention, typeBuilder, out _).CreateType()!;
```

Reply:
Can do, but that was just the dotnet format.